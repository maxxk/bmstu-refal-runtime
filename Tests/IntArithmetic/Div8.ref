$ENTRY Go
{
	= <Prout <Div (1 12312 45145 98981 2394 123 12 0) 73123 231 29>>
}
