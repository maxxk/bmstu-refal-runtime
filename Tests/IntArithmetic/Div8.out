58736 1590576164 737224635 2123290763 3241346931 
